src/lib/loaders/api/loader_without_authentication_factory.js

```diff
@@ -57,30 +56,7 @@ export const apiLoaderWithoutAuthenticationFactory = (
                     time,
                     cache: true,
                   })
-
-                  // Then refresh cache
-                  return throttled(
-                    key,
-                    () => {
-                      api(key, null, apiOptions)
-                        .then(({ body, headers }) => {
-                          verbose(`Refreshing: ${key}`)
-
-                          if (apiOptions.headers) {
-                            return cache.set(key, { body, headers })
-                          } else {
-                            return cache.set(key, body)
-                          }
-                        })
-                        .catch(err => {
-                          if (err.statusCode === 404) {
-                            // Unpublished
-                            cache.delete(key)
-                          }
-                        })
-                    },
-                    { requestThrottleMs: apiOptions.requestThrottleMs }
-                  )
+                  return data
```

Comment:
this is so much cleaner. The only thing I'd do differently is rip the whole cache out, to start - see me larger comment. 

I feel like premature caching is the root of the hardest and most damaging bugs I've worked on.